rfcs/038-sierra-series/README.md

```diff
@@ -0,0 +1,432 @@
+
+As series and Host Item Entries can have volumes, the proposal is to represent them by creating one record for the Series or
+Host Item Entry and one for the volume with a nested `partOf` property (see example below).
+
+Additionally, if a series or Host Item Entry has an identifier (such as ISSN os ISBn or other) the proposal is 
+to mint a canonical identifier in the pipeline and to expose that identifier in the API. 
+Finally, the proposal is to model a series statement or a Host Item Entry with a type `Work`.
+
+### Examples
+ 
+- [1074785](https://search.wellcomelibrary.org/iii/encore/record/C__Rb1074785?lang=eng&suite=cobalt&marcData=Y) 
+This is an example of a series statement in 490 + 830. In this example the information contained in 830 and 490 is 
+  exactly the same, so the result is just one statement. 
+```yaml
+"partOf": [
+  {
+    "title": "Morphogenesis of the vertebrate brain; 2",
+    "workType": {
+      "id": "???",
+      "label": "Volumes",
+      "type": "Format"             
+    },
+    "partOf": [
+      {
+        "title": "Morphogenesis of the vertebrate brain",
+        "type": "Series"
+      }
+    ],
+    "type": "Series"
+  }
+]
+```
+- [3109805](https://search.wellcomelibrary.org/iii/encore/record/C__Rb3109805?lang=eng&suite=cobalt&marcData=Y)
+  490 + 773 with slightly different information
+```yaml
+"partOf": [
+  {   //490
+    "title": "Early European Books : Printed sources to 170; Collection 4",
+    "workType": {
+      "id": "???",
+      "label": "Volumes",
+      "type": "Format"
+    },
+    "partOf": [
+      {
+        "title": "Early European Books : Printed sources to 170",
+        "type": "Series"
+      }
+    ],
+    "type": "Series"
+  },
+  {   //773
+    "title": "Early European Books",
+    "type": "Work"
+  }
+]
+```
+
+- [3178769](https://search.wellcomelibrary.org/iii/encore/record/C__Rb3178769?lang=eng&suite=cobalt&marcData=Y) 490 + 830 multiple entries + 773
+```yaml
+"partOf": [
+  {   // 490 & 830 with same information
+    "title": "Perspectives in Continental philosophy series; no. 39",
+    "workType": {
+      "id": "???",
+      "label": "Volumes",
+      "type": "Format"             
+    },
+    "partOf": [
+      {
+        "id": "abcdefg",
+        "identifiers": [
+          {
+            "identifierType": {
+              "id": "issn",
+              "label": "ISSN",
+              "type": "IdentifierType"
+              },
+            "value": "1089-3938",
+            "type": "Identifier"
+          }
+        ],
+        "title": "Perspectives in Continental philosophy series",
+        "type": "Series"
+      }
+    ],
+    "type": "Series"
+  },
+  {  // Second 830 with slighly different information
+    "title": "Fordham perspectives in continental philosophy",
+    "type": "Series"
+  },
+ {   // 830 & 773 contain the same information
+    "title": "ACLS Humanities E-Book.",
+    "type": "Work" //or Series?
```

Comment:
Just realised that 773 for `ACLS Humanities E-Book.` has URL: http://www.humanitiesebook.org/ in subfield $n and that is shown in Encore, so it should probably made to fit in this model. Not sure where it would fit in the `Work` model though?

Reply:
It would be good to know how many have a $n, but I'm going to go with thinking it's ok to drop this. You have to access and sign in via the library site anyway, so a direct link to the publisher site isn't that useful.

Reply:
If we did have to do this, in theory we could handle it like other resource links, but I have no idea how we would then render it on the page. Given the limited utility anyway, I don't think we should try to shoehorn it in.